src/assets/CloseSVG.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+interface ICloseSVG {
+  closeButtonHandlerByCreateTodo?: () => void;
+  closeButtonHandlerByIsVisible?: () => void;
+  createTodo?: boolean;
+  closeButtonHandlerByCreate?: () => void;
+  closeButtonHandlerBySetting?: () => void;
+  isCreateProject?: boolean;
+  isVisible?: boolean;
+}
+
+const CloseSVG: React.FC<ICloseSVG> = ({
+  closeButtonHandlerByCreateTodo,
+  closeButtonHandlerByIsVisible,
+  closeButtonHandlerByCreate,
+  closeButtonHandlerBySetting,
+  createTodo,
+  isCreateProject,
+  isVisible
+}) => {
+  const closeSvgHandler = () => {
```

Comment:
instead of this, you can put one function onClick in props and just pass needed function in needed place, you will decrease complexity of this logic